src/shared/use-data-value-set/use-data-value-set.js

```diff
@@ -57,7 +57,12 @@ const select = createSelector(
 
 const meta = { persist: true }
 
-export const useDataValueSet = () => {
+// TODO: investigate why the same data is fetch twice.
+// A. When refreshing the page it fires once when the form is rendered and the second
+// time when the form is focussed ()
+// B. When making a new selection it simply fires twice without any user-interaction
+// Also: it would make sense if this would return a 304, but we're getting a 200
```

Comment:
This worries me 😢

Reply:
This is a simple fix. Just needs `refetchOnMount: false` in the `useQuery`-options.

Reply:
I think we can remove this comment since it's fixed in #123 .